test/tests/performance/pbs_history_cleanup_quasihang.py

```diff
@@ -94,7 +94,7 @@ def test_time_for_stat_during_history_cleanup(self):
         now2 = 0
         i = 0
 
-        while (now2 - now1 < 2 and i < 125):
+        while now2 - now1 < 2 and i < 125:
```

Comment:
Thank you for removing the outer pair of ().  Now that I'm looking at this while loop, I'm not 100% sure which is higher precedence between the minus and less than.  Maybe removing them was the wrong idea.  You might want to put the () back around the now2 - now1.  Your choice.

Reply:
let us keep it as it is now. As per the operator precedence in Python, this will do what we desire.